### ISSUE TITLE
Implement PRs CI checks that run on an OpenShift cluster

### ISSUE THREAD

**Issue by fabiobrz** [Intersmash/intersmash#1]
Intersmash should be able to test itself.
This issue specifically tracks the implementation of a mechanism that should run the integration tests on an OpenShift cluster.

The preferred solution would be to evaluate a solution based on the https://docs.ci.openshift.org/docs/ infrastructure.

The CI checks should at least be triggered when submitting a PR or pushing changes to it, while the implementation of a a comment based trigger (e.g.: `/test`) would be nice to have. 

**Alternatives**

* There's a chance to use a Red Hat OCP developer sandbox, see https://developers.redhat.com/developer-sandbox which could serve this issue purpose with a couple of limitations, i.e. it would expire after some time (but a new one can be ready in a few minutes) and would not be such good when it comes to load, but at the moment we're not in a situation where a concerning load is expected.

* Instead, on a bright side, the sandbox is easily usable through GitHub actions, see e.g. https://redhat-scholars.github.io/openshift-starter-guides/rhs-openshift-starter-guides/4.7/nationalparks-java-codechanges-github-actions.html and would allow for this issue to be solved quickly, since _things need to be done_, while a more advanced and suitable solution will be explored.
---
* Environment: GitHub + OpenShift
* Steps to reproduce: N/A

**Comment by fabiobrz** [comment]
Reopening since was automatically closed. Will be closed once https://github.com/openshift/release/pull/37965 is merged.
